### DISCUSSION TITLE
Looks like Postal is installed & running but unable to see Dashboard / Portal

### DISCUSSION THREAD

**Discussion by SFARPak** [postalserver/postal#1958] [Help with using Postal]
![image](https://user-images.githubusercontent.com/89567338/163790134-aded52b3-a0b5-4f35-a206-81592bcb6d8b.png)

Hello Everyone,

Looks like Postal is installed & running but unable to see Dashboard / Portal.

Best Regards

**Comment by willpower232** [comment]
are you running nginx or caddy or something to proxy the UI?

**Reply by SFARPak** [reply]
yes actually I started Caddy for SSL which was mentioned in the instructions.

![image](https://user-images.githubusercontent.com/89567338/164289262-48dd9f55-ee8f-40da-a46f-b04f7a419e59.png)

What should I do now I tried to stop Caddy and restart Postal again but still no dashboard.

**Reply by SFARPak** [reply]
One more thing which might help us resolve this is there are no ports in docker containers for these containers:

![image](https://user-images.githubusercontent.com/89567338/164291109-c4a47911-dceb-4151-96f8-37924cfe192e.png)

**Reply by willpower232** [reply]
the ports probably aren't present because of the host networking, when you say you can't see the UI, what are you seeing?

**Reply by SFARPak** [reply]
The site can't be reached.

I have checked the DNS that is also pointed to the IP.

**Reply by willpower232** [reply]
Is it possible there is a firewall blocking port 80 and/or 443?

**Reply by SFARPak** [reply]
Ohh! Yes I thought I had already opened firewall ports.

But some reason there is issue.

Thank You for help.

I will get back after I resolve ports issue.

**Comment by SFARPak** [comment]
![image](https://user-images.githubusercontent.com/89567338/164980468-c446608e-f923-4a96-af89-713154de79e4.png)

Now I have resolved the port issue and you can see if I start Nginx its working.

But when I stop Nginx postal should show up.

But unfortunately...

**Reply by willpower232** [reply]
that is the default nginx page, have you configured it to proxy port 5000?

**Reply by SFARPak** [reply]
I meant to say the port 443 is open thats why I wanted to show Nginx with SSL.

But when I disable Nginx even though postal portal doesn't show up.

I have port 5000 open as well. Do I need to do anything else to configure it as proxy? Please share some details as I am not good at proxy.

**Reply by johnceci-m3** [reply]
I would recommend maybe taking a step back and think about using Caddy (web server), it is a real simple config file change to setup the reverse proxy and get the dashboard working.  Might be real simple for nginx, but the docs for Postal show you how to do it with Caddy...

**Reply by willpower232** [reply]
Caddy config is much shorter and has less to go wrong but whatever you have used, can you share the config you do have if you're still stuck?

**Comment by alitechio** [comment]
Hello Gents,

Sorry for the delayed response.

I actually tried Nginx before just to see if the port is open 443.

Thats why I shared its screen shot that port is correctly open.

The final thing I had to do is run caddy docker container which I didn't run I thought the server is sidecar to postal but it was not so that was the confusion I had.

Thank you guys now after running Caddy it works like charm !

Appreciate that.
